static/styles/account/settings.scss

```diff
@@ -5,6 +5,15 @@
   padding-top: 20px;
 }
 
+.input-group-btn {
```

Comment:
If the button groups should look the same for the different pages, we should move this style to a more general scss file, so we don't need to repeat it. Maybe `schulcloud-client/static/styles/lib/base.scss` is the right file for this?